CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 - #1778 - Added folder support to system notifications
 - #1780 - Added a new version of the XSS Taglib to support the sling XSSAPI.
 - #1783 - Added the possibility to replace the existing host in an attribute
+- #1795 - Added the Asset Content Packager and unit tests
```

Comment:
Comment should probably omit "and unit tests" - no other features mention that and it's more just a prerequisite of adding the feature in the first place.

Reply:
Updated Changelog